.travis.yml

```diff
@@ -1,7 +1,7 @@
 os: osx
 language: node_js
 before_install:
-  - brew update && brew install libpng
+  - brew update && brew ls --versions libpng || brew install libpng
```

Comment:
I guess this should be removed?

Reply:
As you wish. This is because Travis is broken. You can see the results for each commit.

Reply:
So, should I remove this commit and let Travis fails? Should I create another PR for this? 🤔

Reply:
I'm probably missing something, but can you explain why `brew ls --versions libpng` fixes Travis?

Reply:
> I'm probably missing something, but can you explain why brew ls --versions libpng fixes Travis?

From the [log of the broken job](https://travis-ci.org/tcoopman/image-webpack-loader/jobs/424135266#L635): 
```
Error: libpng 1.6.34 is already installed
To upgrade to 1.6.35, run `brew upgrade libpng`
The command "brew update && brew install libpng" failed and exited with 1 during .
Your build has been stopped.
```

The instruction is `brew install libpng`; is trying to do the installation directly, the error is because that lib is already installed. My solution was install it just if is not available.

`brew ls --versions libpng` prints the installed versions, **returning zero if non is installed**. `brew ls --versions libpng || brew install libpng` could be read then as "_if there is no installed version, install libpng_".

> I've just merged #163, can you check if that changes anything for you?

That is a different approach to avoid the same error. `brew outdated libpng || brew upgrade libpng` is updating the library if is outdated... it is **assuming is installed**, fix Travis because disappears the `brew install`.

At the end, answering the question, yes, it also solves Travis. I will remove that commit :) 👍

Reply:
Done! 👍